nixos/tests/haproxy.nix

```
@@ -43,5 +45,12 @@ import ./make-test-python.nix ({ pkgs, ...}: {
     assert "haproxy_process_pool_allocated_bytes" in machine.succeed(
         "curl -k http://localhost:80/metrics"
     )
+
+    with subtest("reload"):
+        machine.succeed("systemctl reload haproxy")
+        sleep(1)
```

```suggestion
        # wait some time to ensure the following request hits the reloaded nginx
        machine.sleep(5)
```

I propose to even sleep a bit longer.

---

Done.